src/client.js

```diff
@@ -343,17 +396,6 @@ class Client {
     return rawResponse.map(Role.decode)
   }
 
-  /**
-   * Requests the deletion of a named TozID Realm belonging to the account.
-   *
-   * @param {string} realmName The name for the realm to delete.
-   *
-   * @returns {Promise<Object>} Empty object.
-   */
-  async deleteRealm(realmName) {
-    return this.api.deleteRealm(this.queenClient, realmName)
-  }
-
```

Comment:
This method was moved up the file to colocate it with the other realm management code.